unit_test/CMakeLists.txt

```diff
@@ -87,6 +87,58 @@ IF (KOKKOS_ENABLE_CUDA)
   )
 ENDIF ()
 
+IF (KOKKOS_ENABLE_SYCL)
+  KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/sycl)
+  KOKKOSKERNELS_INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_DIR}/sycl)
+
+  APPEND_GLOB(SYCL_BLAS_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/sycl/Test_SYCL_Blas*.cpp)
```

Comment:
There is only one source file in SYCL: `Test_SYCL_Blas.cpp` so I think the APPEND_GLOB should be removed and the `${SYCL_BLAS_SOURCES}` should be replaced with `sycl/Test_SYCL_Blas.cpp`.
This is also the idea behind the refactor of the unit-test, this way developers are using header only tests that automatically get propagated to all the backends.